src/components/HistoryModal/index.js

```diff
@@ -0,0 +1,17 @@
+import {ModalContainer, Modal, StyledHistoryModalHeader, StyledHistoryModalButton} from './styled';
+
+export default function HistoryModal({children, cancelHistoryForm}) {
+	return (
+		<ModalContainer>
+			<Modal open>
+				<StyledHistoryModalHeader>
+					<h2>History</h2>
+					<StyledHistoryModalButton type="button" onClick={cancelHistoryForm}>
```

Comment:
When passing a callback function, usually the prefix `on` is used to distinguish them between all the other props.
```suggestion
export default function HistoryModal({children, onCancelHistoryForm}) {
	return (
		<ModalContainer>
			<Modal open>
				<StyledHistoryModalHeader>
					<h2>History</h2>
					<StyledHistoryModalButton type="button" onClick={onCancelHistoryForm}>
```